itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -3185,6 +3186,27 @@ public void testLoadCmPathMissing() throws IOException {
     fs.create(path, false);
   }
 
+  @Test
+  public void testDumpFileMissing() throws IOException {
```

Comment:
The test name is misleading. It can be "testDumpWithTableDirMissing".

Reply:
done